### PR TITLE
JWWEB-859 Markup for edit Contact item 

### PR DESCRIPTION
**Pull request by Creepypoke**
This PR based on #874

### CODE REVIEW

File: src/pages/ContactsItem/ContactsItem.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import {
+  type Props,
+  ContactsItemView,
+} from 'pages/ContactsItem/ContactsItemView'
+
+type OwnProps = {|
+  +contactId: ContactId,
+|}
+
+function mapStateToProps(state: AppState, { contactId }: OwnProps) {
```
Comment (flexxnn) [inline src/pages/ContactsItem/ContactsItem.js]: We don't need mapStateToProps, witch does nothing

File: src/pages/ContactsItemEdit/ContactsItemEditView.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+import React from 'react'
+import { t } from 'ttag'
+import {
+  Form,
+  Field,
+  type FormRenderProps,
+} from 'react-final-form'
+
+import {
+  Button, JInputField, JLink,
```
Comment (flexxnn) [inline src/pages/ContactsItemEdit/ContactsItemEditView.js]: Lint

File: src/pages/ContactsItem/ContactsItem.js (hunk 1 above)
Comment (Creepypoke) [reply src/pages/ContactsItem/ContactsItem.js]: But it will needed in future

File: src/pages/ContactsItemEdit/ContactsItemEditView.js (hunk 2 above)
Comment (Creepypoke) [reply src/pages/ContactsItemEdit/ContactsItemEditView.js]: ESLint autofix result :)

**Pull request merged**